yoti-sdk-api/src/main/java/com/yoti/api/client/qrcode/QrCode.java

```diff
@@ -0,0 +1,23 @@
+package com.yoti.api.client.qrcode;
+
+/**
+ * Dynamic QRCode for a customised policy for an application
+ *
+ */
+public interface QrCode {
```

Comment:
I've gone with QrCodeResult

Reply:
My point with this is that you are getting a URL that can be displayed in a QR code to trigger a sharing transaction, but it doesn't have to. It could be used for an NFC tap, for instance.